### PR TITLE
Bugfix/wrench offhand fix

### PR DESCRIPTION
**Pull request by jollycar**
When using the wrench to connect nodes AND you have blocks or items in your offhand, the blocks get placed AND the wrench is interacting with the node

fixes https://github.com/Direwolf20-MC/LaserIO/issues/73

### CODE REVIEW

**Comment by jollycar** [comment]
Note: this my first ever pull request into a minecraft project. Feedback is welcome. I hope this will be the first of many.

File: src/main/java/com/direwolf20/laserio/util/VectorHelper.java [hunk 1]
```diff
@@ -12,19 +11,16 @@
  * this means that we can use COLLIDER so it traces through non-collidable objects
  */
 public class VectorHelper {
-    public static BlockHitResult getLookingAt(Player player, ItemStack tool, int range) {
+
+    public static final int RANGE = 10; // How far away you can click on blocks from
```
Comment (jollycar) [inline src/main/java/com/direwolf20/laserio/util/VectorHelper.java:15]: perhaps better to move to a settings/config class or better: configuration file?

**Comment by Direwolf20-MC** [comment]
I'm going to see if I can resolve this one without using a forge event.

**Comment by Direwolf20-MC** [comment]
Resolved by: https://github.com/Direwolf20-MC/LaserIO/commit/e5a9e07c3e17201b3b7bdb773c05f8bd18c09c44 :)

**Pull request closed without merging**